### ISSUE TITLE
It may make sense to upstream your spec into https://github.com/kaitai-io/kaitai_struct_formats/

### ISSUE THREAD

**Issue by KOLANICH** [wavetilt/node-aseprite#1]

**Comment by Qix-** [comment]
Hey! Yes, although I feel as though this should be maintained by the Aseprite team (since we do not make nor maintain Aseprite).

If they would like to use the spec here and take control of the spec, they're absolutely free to! The one problem I think they'll find however is that Kaitai doesn't support generation, which limits its usefulness for a project like Aseprite.

**Comment by KOLANICH** [comment]
>however is that Kaitai doesn't support generation, which limits its usefulness for a project like Aseprite.

Yeah, serlization is not yet implemented. There are some targets that have some elements of that, but in general it is not really implemented, even the base layer.
